src/components/HeroBanner.tsx

```diff
@@ -1,63 +1,58 @@
-import React from 'react';
-import heroBanner from '../assets/img/HeroBanner.png'; // Adjust the path relative to your project structure
-import Button from './Button';
-import { useNavigate } from 'react-router-dom'; // Import useNavigate for navigation
+import React from "react";
+import heroBanner from "../assets/img/hero.png"; // Adjust the path relative to your project structure
+import Button from "./Button";
+import { useNavigate } from "react-router-dom"; // Import useNavigate for navigation
 
 const HeroBanner: React.FC = () => {
   const navigate = useNavigate(); // Get navigate function from React Router
 
   const handlePickup = () => {
-    // Navigate to '/quote-details' when a truck button is clicked
-    navigate('/distance-calculator');
+    // Navigate to '/distance-calculator' when a truck button is clicked
+    navigate("/distance-calculator");
   };
 
   return (
-    <div className="relative flex flex-col md:flex-row justify-center items-center h-screen"
+    <div
+      className="relative flex flex-col md:flex-row justify-between items-start px-6 md:px-12 w-full h-screen"
       style={{
-        backgroundImage: `url(${heroBanner})`,
-        backgroundSize: 'cover',
-        height: '90vh', // Set the height to 90% of viewport height
+        background: "linear-gradient(-180deg, #7783D2 13%, #F0F0F0 83%)",
       }}
     >
+      <div className="text-left md:px-30 lg:px-48 mt-24 md:mt-32 lg:mt-48">
```

Comment:
try to decrease the left padding here to make the whole hero section look balance.